types/hubot/index.d.ts

```diff
@@ -1,17 +1,63 @@
-// Type definitions for hubot 2.19
+// Type definitions for hubot 3.3
 // Project: https://github.com/hubotio/hubot
 // Definitions by: Dirk Gadsden <https://github.com/dirk>
 //                 Kees C. Bakker <https://github.com/KeesCBakker>
 //                 Emil Marklund <https://github.com/eeemil>
+//                 Jon Phenow <https://github.com/jphenow>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
-
-/// <reference types="node" />
+// TypeScript Version: 3.7
 
 import { EventEmitter } from 'events';
+import { Options as HttpOptions, ScopedClient } from 'scoped-http-client';
+import { Server } from 'http';
+import { Express } from 'express';
 
 declare namespace Hubot {
-    class Brain<A> extends EventEmitter {
+    interface UnknownData {
```

Comment:
This doesn't seem to provide a lot of value compared to just using `{}`.

Reply:
Yea it seemed more interesting when I started it. I'll replace the usages.